src/remote/utils.go

```diff
@@ -35,6 +37,24 @@ import (
 // xattrName is the name we use to record attributes on files.
 const xattrName = "user.plz_hash_remote"
 
+func (c *Client) pushTreeDigestDownloadError() {
+	if c.state.Config.Remote.PrometheusGatewayURL == "" {
+		log.Infof("No Prometheus pushgateway URL to push Digest Download error to")
+		return
+	}
+	downloadErrorCounter := prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "please",
+		Name:      "tree_digest_download_eof_error",
+	})
```

Comment:
Is this not meant to be long lived? Creating a new counter every time we push will result in the count always being 1 no?

Reply:
I've tested this some more locally and checked with observability, and it doesn't reset the counter on the gateway. I have moved it though so it's only called when the remote client is initialised, so that we're not calling it for every counter increment.